apps/src/lib/levelbuilder/rubrics/RubricEditor.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LearningGoalItem from './LearningGoalItem';
+import Button from '@cdo/apps/templates/Button';
+
+export default function RubricEditor({
```

Comment:
this looks like some new UI, would you mind please including a screenshot showing what this adds? this isn't strictly necessary on every PR if everyone knows what's going on, but I'm afraid I'm still ramping up on the UI side of this feature